src/external.c

```diff
@@ -73,7 +75,7 @@ uint32_t external_srv_auth(struct gssntlm_ctx *ctx,
     return winbind_srv_auth(cred->cred.external.user.data.user.name,
                             cred->cred.external.user.data.user.domain,
                             ctx->workstation, chal_ptr,
-                            nt_chal_resp, lm_chal_resp, session_base_key);
+                            nt_chal_resp, lm_chal_resp, session_base_key, num_sids, sids);
```

Comment:
Line wrapping.

Reply:
Do you still have line length limit of 80 characters?
Well, let's fix cosmetic issues after we agreed all design items :-)

Reply:
Yes 80 char is our limit